lib/stacks/iam.ts

```diff
@@ -822,25 +834,7 @@ export class IAMStack extends Stack {
                         ],
                         conditions: {
                             StringEquals: {
-                                'iam:PassedToService': 'sagemaker.amazonaws.com',
```

Comment:
Changing this back. This consolidation works, but allows passing a broader spectrum of roles to translate which is less-scoped. If we need to conserve policy space in the future, this could probably be done.